src/component/GenomeNexusLogo.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-import gnLogo from '../image/logo/header_logo_with_white_text.png';
+import gnLogo from '../image/logo/header_logo_with_white_text_small_size_437 × 58.png';
```

Comment:
good to not have spaces in the filename

Reply:
@inodb can't believe there was a very very tiny space in the file name that I overlooked all the time 😂